crates/ruma-events/src/space/child.rs

```diff
@@ -50,6 +51,25 @@ impl SpaceChildEventContent {
     }
 }
 
+/// An `m.space.child` event represented as a Stripped State Event with an added `origin_server_ts`
+/// key.
+#[derive(Clone, Debug, Deserialize, Serialize)]
```

Comment:
I'm a little wary of using the regular `Deserialize` and `Serialize` impls here. We use the `Event` derive for `PresenceEvent`, should be simple enough to adjust it to work here as well.